tools/ucalls.py

```diff
@@ -14,8 +14,20 @@
 
 from __future__ import print_function
 import argparse
-from bcc import BPF, USDT
+from bcc import BPF, USDT, USDTException
 from time import sleep
+import sys
+
+def enable_probe_or_bail(usdt, probe, function):
+    try:
+        usdt.enable_probe(probe, function)
+    except USDTException:
```

Comment:
i think the copy paste avoidance trumps modularization concerns. i think of bcc/usdt.py as bcc specific usdt module.